addons/hr_contract/report/hr_contract_history_report_views.xml

```diff
@@ -74,7 +74,7 @@
                                       decoration-primary="state == 'open'"
                                       decoration-muted="state == 'close'"
                                       decoration-bf="id == parent.contract_id"
-                                      default_order = "date_start DESC, state DESC"
+                                      default_order = "date_start desc, state desc"
```

Comment:
default_order="date_start desc, state desc, id desc"

Reply:
Hello @fmdl !
Thanks, I do understand your point of view on a UX perspective 🙂

Here as we only display not archived contracts and contracts that are not in 'cancel' state, chances are null in real life that you have two contracts for the same employee with the same start_date and the same state. And if it happens, the user should archive or set the state to 'cancel' to one of the two.

Have a great day!